DanielBresnahan1/MDMGAGAICSCS
# Cucumbers and vinegar2

Background: Two changes are implemented in this code: A reset button that clears the cookies from the cache, resulting in the training restarting, and cookies to remember a users progress on the website. 

To test: 
1. Pull down the code and deploy the application.
2. Navigate to the application in a normal tab (not private)
3. Work your way through the application. 
4. After finishing the first round of training, hit the home button, and reenter the training. You should only receive 5 images, and after completing this set, you should have trained off of a total of 15 images. 
5. Hit the reset button. This should redirect you to home.
6. Start the training again. This time, you should be shown ten images, and only those ten images should be shown on the final screen.
7. Once again, his the restart button, and begin the training.
8. Before finishing the first round, hit the home button or leave the website and return.
9. Start the training one final time. This application should operate normally.
